Zend/zend_compile.h

```diff
@@ -483,7 +483,7 @@ union _zend_function {
 		zend_function *prototype;
 		uint32_t num_args;
 		uint32_t required_num_args;
-		zend_arg_info *arg_info;
+		zend_arg_info *arg_info;  /* index -1 —if set— represents the returned value info */
```

Comment:
/* index -1 represents the returned value info
 * if the ZEND_ACC_HAS_RETURN_TYPE flag is set */

Not clear what "if set" means otherwise.

I think it's enough to document this here in zend_function -- if not then it should be in *both* zend_internal_function and zend_op_array, not just one of them.

Reply:
Agree to document only `zend_function`.
I've updated the phrasing of `… —if set— …` to `… if any.`

Reply:
I believe if there is no return type, then the index -1 doesn't exist, right?